app/invoices/edit/template.hbs

```diff
@@ -135,14 +135,13 @@
                       </div>
                     </td>
                     <td class="col-xs-1">
-                      <div class="form-group">
-                        {{select-list
-                          class='form-control'
+                      {{#em-form model=detail submitButton=false as |form|}}
```

Comment:
Does it work well? Because you kinda introduce a form within a form.

Reply:
It seems to work the same as before. One thing is you have to define "model=detail" inside the {{#em-form}}, and the form.select uses that to find its property. So I'm not sure if it can work if I define the {{#em-form}} outside of that {{#each item.details as |detail|}} block, though I can try it.

Reply:
alright. then I make the PR to "require changes"